lib/templates.js

```diff
@@ -238,9 +238,48 @@ function _getPackageID(namespace, packageIds, resourceType) {
     return packageIds.find((id) => id.includes("-" + resourceType)) || packageIds.find((id) => id.includes(namespace));
 }
 
+const createIntegrationDependencyTemplate = (serviceName, serviceDefinition, appConfig, packageIds) => {
+
+    const ordExtensions = readORDExtensions(serviceDefinition);
+
+    const events = Object.keys(serviceDefinition.model.definitions)
+        .filter(event => {
+            const definition = serviceDefinition.model.definitions[event];
+            return definition["@topic"] && definition["@topic"].startsWith(serviceName) && !event.startsWith(serviceName);
+        })
+        .map(event => ({ eventType: serviceDefinition.model.definitions[event]["@topic"] }));
+
+    return {
+        ordId: `${appConfig.ordNamespace}:integrationDependency:${serviceName}:v1`,
+        title: ordExtensions.title ?? serviceDefinition['@title'] ?? "Integration Dependency sample title",
```

Comment:
In general, could you extend your test case with all `??` in this function? I only see the default value